assets/interfaces/configs.ts

```diff
@@ -79,11 +79,19 @@ export interface IAgendaPreviewConfig extends IPreviewConfig {
     coverage_metadata_fields?: Array<string>;
 }
 
+export type AgendaFilterTypes = 'item_type' | 'calendar' | 'location' | 'region' | 'coverage_type' | 'coverage_status';
+
+export interface ItemTypeFilterConfig {
+    events_only?: boolean;
+    planning_only?: boolean;
+}
+
 export interface IAgendaUIConfig extends IBaseUIConfig {
     _id: 'agenda';
     open_coverage_content_in_same_page?: boolean;
     subnav?: {
-        filters?: Array<string>;
+        filters?: Array<AgendaFilterTypes>;
```

Comment:
It might be good if we were to have these different UI configs documented somewhere, that way PMs are able to get familiar with the different configurations available.

Reply:
added some initial docs, will do the setup on readthedocs later